pkg/network/server.go

```diff
@@ -277,6 +277,19 @@ func (s *Server) handleBlockCmd(p Peer, block *core.Block) error {
 	return s.bQueue.putBlock(block)
 }
 
+// handlePing processes ping request.
+func (s *Server) handlePing(p Peer, ping *payload.Ping) error {
+	if err := p.WriteMsg(NewMessage(s.Net, CMDPing, ping)); err != nil {
```

Comment:
I thought that you need to respond with `pong` for a `ping`, but not send a new ping in response. Otherwise this process will never end.

Reply:
ok, will check this behavior. Here was trying to mimic C# code